test/sqlite/sqllogic_test_runner.cpp

```diff
@@ -23,13 +23,55 @@ SQLLogicTestRunner::SQLLogicTestRunner(string dbpath) : dbpath(std::move(dbpath)
 	config->options.allow_unredacted_secrets = true;
 	config->options.load_extensions = false;
 
+	auto &test_config = TestConfiguration::Get();
+	loading_mode = test_config.GetExtensionLoadingMode();
+	install_mode = test_config.GetExtensionInstallMode();
+
+	config->options.autoload_known_extensions = false;
+	config->options.autoinstall_known_extensions = false;
+	config->options.allow_unsigned_extensions = true;
+	local_extension_repo = "";
+
 	auto env_var = std::getenv("LOCAL_EXTENSION_REPO");
```

Comment:
Ideally the `LOCAL_EXTENSION_REPO` environment variable should also be removed and everything should be moved to the new config